.ci_support/linux_python3.7.yaml

```diff
@@ -23,11 +23,11 @@ gsl:
 hdf5:
 - 1.10.5
 libblas:
-- 3.8 *openblas
+- 3.8 *netlib
```

Comment:
@isuruf is it somehow possible to restrict *openblas, so the rerendering doesn't add *netlib again and again. Building with *openblas allows using netlib or openblas at runtime, but using *netlib for the build somehow leads to a crash. I cannot find the corresponding issue right now, but this was related to a missing symbol in liblapack.